assets/js/base/components/drawer/style.scss

```
@@ -18,7 +21,7 @@ $drawer-width-mobile: 100vw;
 	}
 
 	to {
-		transform: translateX(-$drawer-width);
+		transform: translateX(var(--neg-drawer-width));
```

Same change here:
```suggestion
		transform: translateX(max(-100%, var(--neg-drawer-width)));
```

Then, I think we can completely get rid of `$drawer-width-mobile`? :thinking:

---

Thanks for working on this 🙏 

Because we can now remove [this](https://github.com/woocommerce/woocommerce-blocks/pull/8930/files#diff-f9b64c1414be300a9905e1f33cf5318c3b6356f884363795dccae208f61e43c6R82) and [this](https://github.com/woocommerce/woocommerce-blocks/pull/8930/files#diff-f9b64c1414be300a9905e1f33cf5318c3b6356f884363795dccae208f61e43c6R28), right? 🤔

---

Yes, I think so. It should be possible to remove those.